pam-OSC.lua

```diff
@@ -210,6 +211,20 @@ local function main()
                         '"')
             end
         end
+        
+        -- Send Timecode
+        local slots = Root().TimecodeSlots
+        	
+        for _, slot in pairs(slots:Children()) do
+        	local time = slot.timestring
+        	
+        	if oldTimecodes[slot.no] ~= time or oldTimecodes[slot.no] == nil or forceReload == true then
+        		oldTimecodes[slot.no] = time
+        			
+        		Cmd('SendOSC ' .. oscEntry .. ' "/Timecode' .. slot.no .. ',s,' .. time .. '"')
```

Comment:
Why is it now required to have Send enabled in the OSC Out ?

Reply:
See my comment [here](https://github.com/xxpasixx/pam-osc/pull/8/files/22b3a0fec78182bb8d0bb412fbbc79b8854aa223#r1638309970). The code uses some of the OSC commands sent by MA in order to figure out whether a timecode slot is running or not. This is used to sync the state when the play/pause button is pressed within MA instead of the MIDI controller